test/test.js

```diff
@@ -451,14 +451,41 @@ suite('Atomic HTTP redirect handling', function() {
 suite('credentials mode', function() {
   var omitSupported = !self.fetch.polyfill
 
+  suite('clear cookies', function() {
+    test('clear cookie', function() {
+      return fetch('/cookie?name=foo&value=reset', {credentials: 'same-origin'});
+    })
+  })
```

Comment:
Are the other tests depending on this running first to clear the cookies?

If so, we might want to do it as a suite setup function so it will always run first even if individual tests are ran standalone.

I think you basically do

``` js
setup(function() {
  return fetch(...)
})
```